packages/harpguru-core/src/components/option-item/option-item.tsx

```diff
@@ -11,6 +11,7 @@ export type OptionItemProps<T> = Pick<
   'value' | 'twoColumns'
 > & {
   readonly isSelected: boolean
+  readonly isMultiSelect?: boolean
```

Comment:
If we're going to have optional params here then arguably the `twoColumns` param can be optional too.

Reply:
Probably a good point, but not for immediate attention. Issue raised: https://github.com/js-jslog/harpguru/issues/92